pipenv/shells.py

```diff
@@ -228,12 +228,19 @@ def _detect_emulator():
         keys.append("cmder")
     if os.environ.get("MSYSTEM"):
         keys.append("msys")
+    # VS Code internal terminal support
```

Comment:
I don't have a way to test this.   If its still required lets do it in a separate PR for VS code terminal support.